boards/pic32-wifire/include/board.h

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright 2016, Imagination Technologies Limited and/or its
+ *                 affiliated group companies.
+ */
+
+/**
+ * @defgroup    boards_pic32-wifire Digilent PIC32 WiFire
+ * @ingroup     boards
+ * @brief       board configuration for the Digilent PIC32 WiFire
+ * @details
+ * See:
+ * http://store.digilentinc.com/chipkit-wi-fire-wifi-enabled-mz-microcontroller-board/
+ * for more information on the board.
+ *
+ * @{
+ *
+ * @file
+ * @brief       board configuration for the Digilent PIC32 WiFire
+ *
+ * @author      Imagination Technologies.
+ */
+
+#ifndef _BOARD_H_
+#define _BOARD_H_
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+#include "p32mz2048efg100.h"
+
+/**
+ * @brief The peripheral clock is required for the UART Baud rate calculation
+ *        It is configured by the 'config' registers (see pic32_config_settings.c)
+ */
+#define PERIPHERAL_CLOCK (100000000)  /* Hz */
```

Comment:
For consistency over all boards, I would expect this kind of define in the `periph_conf.h` file...